test/kitchen/kitchen.vagrant.yml

```diff
@@ -19,6 +19,10 @@ platforms:
       volumes:
         - /sys/fs/cgroup:/sys/fs/cgroup:ro # required by systemd
 
+  - name: centos-6
```

Comment:
CentOS 6 EOL is 11/30/2020 -- I'm not sure adding this to the test matrix would be beneficial at this juncture.

Reply:
As above.